codegen/model.py

```diff
@@ -823,6 +835,54 @@ def codegen(
         return outputs
 
 
+class MistralCode(VLlmDecoder):
+    def __init__(self, name: str, **kwargs) -> None:
+        kwargs["conversational"] = True
+        super().__init__(name, **kwargs)
+        self.eos += ["\n```"]
+
+    def codegen(
+        self, prompt: str, do_sample: bool = True, num_samples: int = 200
+    ) -> List[str]:
+        prompt = f"""Below is an instruction that describes a task. Write a response that appropriately completes the request.
+{prompt}

```

Comment:
wrap the prompt with python code block. start the reply with python block as well.

can follow the example here: https://github.com/evalplus/evalplus/blob/af3f27923d6ee7ac2100b24e62eab03708e5456a/codegen/model.py#L272